### PR TITLE
chore: enable contracts that remove a-instructions

### PR DESCRIPTION
**Pull request by ashuralyk**

### CODE REVIEW

**Comment by ashuralyk** [comment]
@Flouse ready to review

File: crates/relayer/Cargo.toml [hunk 1]
```diff
@@ -105,7 +105,7 @@ jsonrpc-core = "18.0"
 strum = { version = "0.24.1", features = ["derive"] }
 lazy_static = "1.4.0"
 
-ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "a3c3c3a2" }
+ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "ac9850ff" }
```
Comment (Flouse) [inline crates/relayer/Cargo.toml]:
```suggestion
ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "adb8bcfb033d111174f06b88609aded5b9f2a181" }
```

Comment (Flouse) [reply crates/relayer/Cargo.toml]: - ref: https://github.com/synapseweb3/ckb-ics/pull/30

File: tools/ibc-test/Cargo.toml [hunk 2]
```diff
@@ -28,14 +28,14 @@ jsonrpc-core = "18.0"
 futures = "0.3.27"
 serde_json = { version = "1", default-features = false }
 serde = { version = "1.0", default-features = false }
-ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "a3c3c3a2" }
+ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "ac9850ff" }
```
Comment (Flouse) [inline tools/ibc-test/Cargo.toml]:
```suggestion
ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "adb8bcfb033d111174f06b88609aded5b9f2a181" }
```

File: tools/ibc-test/Cargo.toml [hunk 3]
```diff
@@ -45,7 +45,7 @@ toml_edit = "0.19.14"
 lazy_static = "1.4.0"
 ethers = { version = "2.0.2", features = ["rustls", "ws"] }
 
-forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "ef46058" }
+forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "9971868" }
```
Comment (Flouse) [inline tools/ibc-test/Cargo.toml]:
```suggestion
forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "59d5e92ee3efe2e260bbbbcba2b926e79b063dd2" }
```

- ref: https://github.com/synapseweb3/forcerelay-ckb-sdk/pull/43

**Review by Flouse — APPROVED** [review]

**Pull request merged**